Readme.md

```diff
@@ -58,19 +92,21 @@ When editing content that hasn't been migrated, you may see the following patter
 ```jsx
 import js from '/src/fragments/lib/datastore/js/conflict.mdx';
 
-<Fragments fragments={{ javascript: js }} />;
+<Fragments
+  fragments={{ javascript: js, angular: js, nextjs: js, react: js, vue: js }}
+/>;
 ```
 
 This pattern incorporates fragment files into a page and conditionally renders content based off selected platform added as a condition to the `Fragments` tag.
 
-This fragment would exist in: `pages/src/fragments/lib/datastore/js/conflict.mdx`
+This fragment would exist in: `src/fragments/lib/datastore/js/conflict.mdx`
 
 ### Inline Filters
 
 We are incorporating the use of `<InlineFilters>` to add platform-specific content within the context of one page rather than in fragments. These filters allow you to still specify content by platform and they reference platforms using the same naming convention as our fragments. You can enclose your platform-specific content by updating the opening tag:
 
 ```md
-<InlineFilter filters={["javascript", "react-native", "android", "swift", "flutter"]}>
+<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue"]}>
```

Comment:
Before we showed all options - do we still want to show them like we do in the header example above and just reiterate that they edit out what they aren't using.
```suggestion
<InlineFilter filters={["javascript", "react-native", "angular", "nextjs", "react", "vue", "android", "swift", "flutter"]}>
```